packages/core/src/components/card/card.tsx

```diff
@@ -21,7 +21,7 @@ export interface ICardProps extends IProps, HTMLDivProps {
 
     /**
      * Whether the card should respond to user interactions. If set to `true`,
-     * hovering over the card will increase the card's elevation
+     * hovering over the card will increase the card's elevation to the maximum
```

Comment:
if my suggestion in the other comment sounds good to you, then we don't even need to change this file